deploy/charts/approver-policy/templates/crds/policy.cert-manager.io_certificaterequestpolicies.yaml

```diff
@@ -280,13 +280,11 @@ spec:
                         additionalProperties:
                           type: string
                 selector:
-                  description: Selector is used for selecting over which CertificateRequests this CertificateRequestPolicy is appropriate for and so will used for its evaluation.
+                  description: Selector is used for selecting over which CertificateRequests this CertificateRequestPolicy is appropriate for and so will used for its approval evaluation.
                   type: object
-                  required:
-                    - issuerRef
                   properties:
                     issuerRef:
-                      description: "IssuerRef is used to match this CertificateRequestPolicy against processed CertificateRequests. This policy will only be evaluated against a CertificateRequest whose `spec.issuerRef` field matches `spec.selector.issuerRef`. CertificateRequests will not be processed on unmatched `issuerRef`, regardless of whether the requestor is bound by RBAC. Accepts wildcards \"*\". Nil values are equivalent to \"*\", \n The following value will match _all_ `issuerRefs`: ``` issuerRef: {} ``` \n Required field."
```

Comment:
Are `*` characters supported anywhere in the namespace name? What about multiple wildcards? (e.g. `*-prod-*`)?

Also here you mention `nil values` - but from an end-user of the API perspective it isn't clear what "Nil" means IMO - can we change this to be `the empty value ("")` or something like that?

Reply:
Further to that, if they _are_ meant to be supported anywhere in the namespace name, can we get corresponding test cases?

Reply:
> Are * characters supported anywhere in the namespace name? What about multiple wildcards? (e.g. *-prod-*)?

Yes, [see](https://cert-manager.io/docs/projects/approver-policy/#allowed) 

```
Allowed string fields accept wildcards "*" within its values. Wildcards "*" in
patterns represent any string that has a length of 0 or more. A pattern
containing only "*" will match anything. A pattern containing `"*foo"` will
match `"foo"` as well as any string which ends in `"foo"` (e.g. `"bar-foo"`). A
pattern containing `"*.foo"` will match `"bar-123.foo"`, but not `"barfoo"`.
```

> Also here you mention nil values - but from an end-user of the API perspective it isn't clear what "Nil" means IMO - can we change this to be the empty value ("") or something like that?

Yep agree :+1:

Reply:
> Further to that, if they are meant to be supported anywhere in the namespace name, can we get corresponding test cases?

Sure, more than happy to add more test cases for this and will, but will mention that the actual wildcard implementation is already well tested. https://github.com/cert-manager/approver-policy/blob/main/pkg/internal/util/wildcard_test.go